pkg/vsphere/toolbox/backdoor.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
```

Comment:
Ah right, the code in sigma/bdoor is unix only.  I've added a tag for now so backdoor.go isn't built on windows:

```
// +build !windows
```

We can look at porting bdoor to windows in the future and/or use Go asm instead of cgo.